recipes/libvpx/all/conanfile.py

```diff
@@ -181,16 +174,24 @@ def _execute_configure(self):
             os_name = 'solaris'
         elif host_os == 'Android':
             os_name = 'android'
-        target = f"{arch}-{os_name}-{compiler}"
-        configure_args.append(f"--target={target}")
+        args["--target"] = f"{arch}-{os_name}-{compiler}"
         if str(self.settings.arch) in ["x86", "x86_64"]:
             for name in self._arch_options:
                 if not self.options.get_safe(name):
-                    configure_args.append(f"--disable-{name}")
+                    args[f"--disable-{name}"] = ""
 
-        configure_command = " ".join(configure_args)
-        self.output.info(configure_command)
-        self.run(configure_command)
+        # NOT USED # autotools.configure() # FIXME can use this if we can remove --host and --build flags
+        # libvpx's configure does not understand host and build flags
+        args["--host"] = None
+        args["--build"] = None
+
+        tc.update_configure_args(args)
```

Comment:
required_conan_version should be >=1.57.0

Reply:
done